### DISCUSSION TITLE
Request for new source: https://www.monaloga.de/mportal/awista-logistik/stadt-remscheid

### DISCUSSION THREAD

**Discussion by froggi1930** [mampfes/hacs_waste_collection_schedule#2155] [Ideas]
Good day,

besides TBR there is a second provider for Remscheid. Is it possible to integrate that one too?

[https://www.monaloga.de/mportal/awista-logistik/stadt-remscheid/index.php](https://www.monaloga.de/mportal/awista-logistik/stadt-remscheid/index.php)

Regards,

Georg

**Comment by 5ila5** [comment]
moved to #2156

**Reply by 5ila5** [reply]
UPDATE:

Added in #2160
This will be part of the next release or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

**Reply by froggi1930** [reply]
Thanks a lot for your fast help!
